modules/dedicated-cluster-install-deploy.adoc

```diff
@@ -73,8 +73,9 @@ the *Status* column for any errors or failures.
 
 .. Click the installed *Red Hat OpenShift Logging* Operator.
 
-.. Under the *Overview* tab, click *Create Instance* . Paste the following YAML
```

Comment:
Need verification that this is correct for 4.7. It appears to have been changed in: https://github.com/openshift/openshift-docs/pull/26271/files

Reply:
Leaving comments in place. These files appear to be unused, but I'm going to continue with the CP for consistency with main.